props/saber_chris_buttons.h

```diff
@@ -0,0 +1,1036 @@
+Skip forward 5 presets	Fast double-click AUX, hilt pointing upwards.
+Skip back 5 presets		Fast double-click AUX, hilt pointing downwards.
+Skip forward 10 presets	Fast triple-click AUX, hilt pointing upwards.
+Skip back 10 presets	Fast triple-click AUX, hilt pointing downwards.
+Play Character Quote	Fast double-click MAIN, pointing up. Quotes play sequentially.
+							To reset sequential quotes, change font or run BladeID.
+Play Music Track	    Fast double-click MAIN, pointing down.
+Speak battery voltage	Fast four clicks MAIN.
+Run/Toggle BladeID		Fast triple-click MAIN. (Applicable installs only).
+Enter/Exit VOLUME MENU 	Hold MAIN then quickly click AUX
+						and release both buttons simultaneously.
+Volume up               	Click MAIN while in VOLUME MENU, hilt pointing up.
+Volume down					Click MAIN while in VOLUME MENU, hilt pointing down, or
+							click AUX while in VOLUME MENU.
+							Volume adjusts in increments per click.
+							You must exit VOLUME MENU to resume using saber normally.
+							
+FUNCTIONS WITH BLADE ON
+Blade lockup			Press and hold AUX.
+Blade tip drag			Press and hold AUX while blade is pointing down.
+Force Effect (Quote)	Fast double-click MAIN, hilt pointing down.
+							Plays random Force effect (usually character quote).
+Character Quote			Fast double-click MAIN, hilt pointing up.
+							Plays sequential character quote.
+Lightning block			Double-click MAIN and hold.
+Melt         			Hold MAIN and push blade tip against wall (clash).
+Blaster blocks			Short click AUX.
+Enter multi-blast mode	Hold MAIN while swinging for one second and release. 
+							Saber will play two quick blasts confirming mode.
+							Swing blade to trigger blaster block.
+							To exit multi-blast mode, fast single click AUX.
+
+COLOUR CHANGE FUNCTIONS WITH BLADE ON
+Enter/Exit COLOUR MENU	Hold MAIN then quickly click AUX and release both
+						buttons simultaneously. Or fast triple-click MAIN.
+							Announcement confirms you are in the COLOUR MENU.
+Cycle to next colour    Rotate hilt whilst in COLOUR MENU until desired colour is reached.
+							Most presets have 12 colour options.
+Alt Exit COLOUR MENU	Hold MAIN and wait. 
+							Announcement confirms you are exiting COLOUR MENU.
+							You must exit COLOUR MENU to resume using lightsaber normally.
+
+							
+===========================================================		
+========================= DEFINES =========================
+#define FAKE_BLADE_ID
+Replaces regular BladeID and allows toggling between two different 
+blade/preset arrays regardless of actual BladeID status.
+
+#define SA22C_NO_LOCKUP_HOLD	
+Applicable to two-button mode only, reverts to lockup being triggered
+by clash while holding aux.
+
+GESTURE CONTROLS
+There are four gesture types: Twist, Stab, Swing and Thrust.
+Gesture controls bypass all preon effects.
+Below are the options to add to the config to enable the various gestures. MM.
+#define SA22C_TWIST_ON
```

Comment:
These defines will need to be renamed since they apply to this prop, not SA22C now.